### ISSUE TITLE
Explicit instructions for windows testers

### ISSUE THREAD

**Issue by mjptak** [learningequality/ka-lite#2960]
I'm not sure but the instructions here did not all gel with my attempt to install the ka-lite-testing-0.13 branch.  I think it is because I am working on  and inferior (:)) operating system on this machine.

Here are the instructions I followed:

git clone https://github.com/learningequality/ka-lite.git ka-lite-testing-0.13

cd ka-lite-testing-0.13

git checkout develop

bin/kalite manage setup

bin/kalite manage unpack_assessment_zip http://eslgenie.com/media/assessment_item_resources.zip

bin/kalite start

I think the refinement needs to occur at the bin/kalite manage setup stage but of course have little to base this on.

**Comment by mjptak** [comment]
I think somewhere in the process my $Path variable got truncated.  I probably did something I shouldn't have along the way.

**Comment by MCGallaspy** [comment]
@mjptak so at what step did the installation fail? What's the output, for example, of `bin/kalite start`?

And what's going on with your `path` variable?

**Comment by mjptak** [comment]
As for the path variable somehow (it might not have been the script) the path to python and git were left off.  After I reinserted them early in the path variable using the environment variables interface I re-ran it and made more progress but it got wierd after the ascii KA Lite Logo as it is starting to initialize the database.  It came back to the c prompt but continued like it was looking for input.  I captured it in a txt file but git doesn't like those so I excerpt here.   

![kal1](https://cloud.githubusercontent.com/assets/856425/6113104/54515ab4-b05a-11e4-977e-029862639886.PNG)
